src/modules/layout/components/side-menu/index.tsx

```diff
@@ -59,7 +60,7 @@ const SideMenu = ({ regions }: { regions: HttpTypes.StoreRegion[] | null }) => {
                   >
                     <div className="flex justify-end" id="xmark">
                       <button data-testid="close-menu-button" onClick={close}>
-                        <XMark />
+                        <X />
                       </button>
                     </div>
                     <ul className="flex flex-col gap-6 items-start justify-start">
```

Comment:
Items in the side menu have black color text which doesn't contrast well with the background. Should be white.